phpBB/install/convertors/index.htm

```diff
@@ -0,0 +1,10 @@
+<html>
+<head>
+<title></title>
+<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1">
+</head>
+
+<body bgcolor="#FFFFFF" text="#000000">
+
```

Comment:
Deprecated.
```suggestion
<body style="background-color: #ffffff; color: #000000;">

```

Reply:
Is a copy/paste of the index.htm in /ext

Reply:
Yeah, all of those .htm files are old and deprecated, including the file extension.